### ISSUE TITLE
My Pesterchum is refusing to open

### ISSUE THREAD

**Issue by EssayOfThoughts** [illuminatedwax/pesterchum#25]
I've tried several times and my Pesterchum isn't opening. It shut down unexpectedly and eventually allowed me back in with my original chumhandle and then i closed it properly and now it will not let me on.

**Comment by illuminatedwax** [comment]
Your profile data might be corrupted. Run the debug build and post the
contents of debug.log on pastebin.com, and send us the link
On Jan 26, 2012 2:01 PM, "demonaikaDruidess" <
reply@reply.github.com>
wrote:

**Comment by EssayOfThoughts** [comment]
I am aware this makes me sound stupid, but how do i run the debug build?
Indeed what is a debug build?

**Comment by illuminatedwax** [comment]
Check out the PC thread on the MSPA Forums; you can download the
program there and run it.

On Thu, Jan 26, 2012 at 2:14 PM, demonaikaDruidess
reply@reply.github.com
wrote:

**Comment by EssayOfThoughts** [comment]
Okay i cannot run the MANIFEST file but the pesterchum_debug application opened a window with the text ' Error loading Python DLL: python26.dll (error code 126)'

I hope this is helpful but i am quite inept with computers so i don't know.

**Comment by ghost** [comment]
I get the exact same message as that,
I've also tried re-installing before, but it doesn't work.
After a bit of searching I happened upon a thread where someone said that deleting everything related and then re-installing worked, but I'd rather not delete my information until someone confirms this.
I hope this problem is resolved soon enough because pesterchum is just such a fun program.

**Comment by EssayOfThoughts** [comment]
My laptop died lately and i reinstalled pesterchum and now it works fine, so i think deleting the lot and restarting from scratch with pesterchum is a viable option.

**Comment by ghost** [comment]
but just a question, was anyone able to simply reinstall and have it work?
since the only ways of fixing that I've heard is to delete everything related to it and then reinstalling

**Comment by EssayOfThoughts** [comment]
well i managed to get my pesterchum to work after my laptop crashed and i had to reinstall everything. but it did work by doing that- its all fine now

**Comment by TheDudeWhoDoesntCode** [comment]
Yeah but I have this same problem. I don't wanna delete literally everything I have. It's like buying a new computer....Then again, I don't have much downloaded. No, but still.
Seriously, I even copied the zip one there, put one on desktop, etc. I deleted all of them except for the original (the one I copied from) in the original area. Not only that, I uninstalled and reinstalled. I could go on about the hundred things I tried, moving, copying, pasting, deleting, installing, uninstalling, installing some more, scans, etc. I went through too much to get this working.
Im about to give up.

**Comment by illuminatedwax** [comment]
Try running the debug version and sending me the resulting log!
Instructions are in the MSPA Forum thread. It should let me know exactly
what's wrong so I can help fix it.

**Comment by TheDudeWhoDoesntCode** [comment]
Oh yay, now we can see why it does this!

**Comment by TheDudeWhoDoesntCode** [comment]
It's like karkat. I tried to learn coding once and I only jacked up some of my things. XD It was actually Minecraft where I attempted this.

**Comment by TheDudeWhoDoesntCode** [comment]
SO I'm not quite sure about half of what is really going where sometimes.

**Comment by TheDudeWhoDoesntCode** [comment]
But imma try to run the debug.

**Comment by TheDudeWhoDoesntCode** [comment]
Nothing, not even that one opens up.

**Comment by TheDudeWhoDoesntCode** [comment]
I mean. Well I got nothing in the log.

**Comment by arcaneAgilmente** [comment]
PasteBin the log and link it here.
